doc/api/querystring.md

```diff
@@ -13,6 +13,56 @@ query strings. It can be accessed using:
 const querystring = require('querystring');
 ```
 
+## querystring.decode(str[, sep[, eq[, options]]])
+<!-- YAML
+added: v0.1.25
```

Comment:
These have been added in v0.1.99 (see https://github.com/nodejs/node/commit/4ce100fa622d97de51d7cccde2301653ee95bbe7).

Reply:
Thank you ~